app/services/report_importer.rb

```diff
@@ -95,7 +83,11 @@ def import_log_messages
     end
   end
 
-  def inspect_report
+  def report_status
+    ConfigReportStatusCalculator.new(:counters => raw['status']).calculate
```

Comment:
I'm not sure a deprecation warning is useful, as we can't later remove the method without breaking the tests.  Unless you want it just to return an empty hash always - but it seems more useful I think to tell subclasses to define it by raising an error.

Can you define a subclass of ReportImporter in the test, like you've already done for Report, which defines the method sufficiently for the test to pass?  The method in the base class can safely raise an error then.